whisper:
  ros__parameters:
    # whisper
    model_name: "tiny.en" # other models https://huggingface.co/ggerganov/whisper.cpp
    language: "en"
    n_threads: 4
    print_progress: false

    # buffer
    batch_capacity: 6 # seconds
    buffer_capacity:  2 # seconds
    carry_over_capacity: 200 # milliseconds
